test/fixtures/users.yml

```diff
@@ -0,0 +1,31 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  first_name: MyString
+  ,: MyString
+  last_name: MyString
+  ,: MyString
+  email: MyString
+  ,: MyString
+  user_name: MyString
+  ,: MyString
+  encrypted_password: MyString
+  ,: MyString
+  phone: MyString
+  ,: MyString
+  role: 1
+
+two:
+  first_name: MyString
+  ,: MyString
+  last_name: MyString
+  ,: MyString
+  email: MyString
+  ,: MyString
+  user_name: MyString
+  ,: MyString
+  encrypted_password: MyString
+  ,: MyString
+  phone: MyString
+  ,: MyString
+  role: 1
```

Comment:
I guess this is also not necessary as well, could you please check?

Reply:
deleted. thank you